README.md

```diff
@@ -1,55 +1,70 @@
 # cfgov-refresh
 
-This repository contains the redesign-in-progress of consumerfinance.gov. This includes front-end assets and build tools, and configuration for [Sheer](https://github.com/cfpb/sheer) to load content from Wordpress and Django back-ends to elasticsearch to render the site.
+**Description**: This repository contains the redesign-in-progress of the [consumerfinance.gov](http://consumerfinance.gov) website. This includes front-end assets and build tools, and configuration for [Sheer](https://github.com/cfpb/sheer) to load content from WordPress and Django back-ends to Elasticsearch to render the site.
 
-### This project is a work in progress
+**Technology stack**:
```

Comment:
I feel like the Project Template could provide better guidance on this actually, as the tech stack and dependencies sections in the template appear to overlap each other.

I interpreted the "tech stack" at the beginning of the Readme as system-level environment dependencies (OS, package managers), and the "dependencies" section as being for project-level environment dependencies (which likely are installed within or use the system-level dependencies). Under that premise, node actually should move to the tech stack.

It seems organizationally helpful to list the dependencies before the instructions for installing them, so having sheer and Elasticsearch immediately preceding the instructions for installing them seemed to me to be a good idea.

Homebrew I only included because it's a step in the elasticsearch installation instructions. If removed, that section should probably be re-written to just include a side note of how to find the config file with `brew`.

Perhaps we should merge the tech stack into the dependencies section and open an issue on the template repo calling attention to their ambiguity.